devel/notcurses/Portfile

```diff
@@ -40,4 +40,6 @@ depends_lib-append  path:lib/libavcodec.dylib:ffmpeg \
                     port:ncurses \
                     port:zlib
 
+patchfiles          patch-excise-filesystem-include.diff
```

Comment:
Looks like this patch needs to be added to your commit.

Reply:
[hangs head] i'd killed the file somehow, too. recreated, added, force-pushed, sorry about that.

Reply:
No worries Nick, just trying to help save you time!